src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -63,7 +63,7 @@ export const Details: FC<DetailsProps> = ({ conversation, ...props }) => {
       width={
         props.showDetails ? "376px" : ["376px", "376px", "376px", "376px", "0"]
       }
-      opacity={props.showDetails ? 1 : 0}
+      opacity={props.showDetails ? 1 : (0 as any)}
```

Comment:
TBH, I have no idea what this one was about. I'm assuming some interaction with the manually applied props of `DetailsContainer`. I think this interface is a little weird though as we probably shouldn't be manually doing this: either mixin the opacity system function or this prop should have some kind of semantic meaning. But this is an aside.